src/App.js

```diff
@@ -35,21 +37,76 @@ const styles = theme => ({
 });
 
 class App extends Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      authContext: {
+        authData: {},
+        isAuthenticated: this.isAuthenticated,
+        setCurrentUser: this.setCurrentUser,
+        setCurrentUserProfile: this.setCurrentUserProfile
+      }
+    };
+  }
+
+  componentDidMount = () => {
+    // Sync auth session stored in the localStorage with Context
+    // TBH, I'm not sure if we even need context
```

Comment:
We need a way to notify components that the value has changed. That's what we're using context for

Reply:
I have provided a more detailed response to this question in another comment now in [this comment](https://github.com/codefordenver/members/pull/278#issuecomment-444757012)